### DISCUSSION TITLE
Cannot access git submodules

### DISCUSSION THREAD

**Discussion by mikeboensel** [alefragnani/vscode-bookmarks#590] [General]
I saw from an earlier question that you had set them private. Do you ever plan on opening these up? Without those repos people cannot contribute.

fatal: repository 'https://github.com/alefragnani/vscode-bookmarks-core.git/' not found

**Comment by alefragnani** [comment]
Hi @mikeboensel ,

Yes, I plan to open as much as possible to allow anyone to contribute. I just didn't have enough free time to focus on this topic yet. You may subscribe to #523 to be notified about updates. 

Hope this helps
